### DISCUSSION TITLE
参考文献列表中作者名全是大写字母问题

### DISCUSSION THREAD

**Discussion by liuxybeta** [stone-zeng/fduthesis#256] [Q&A] [answered]
求救，参考文献列表中那些英文文献的作者名字母全是大写，有没有办法改成作者名仅首字母大写？还是大家都默认英文人名所有字母都大写？
![微信截图_20220511202709](https://user-images.githubusercontent.com/66198221/167849644-4c26a188-5adb-43f2-8d54-804f8eb4fca7.png)

**Comment by tanukihee** [comment, marked as answer]
国标规定如此

**Reply by liuxybeta** [reply]
谢谢。那就是不用改对吧，保持全部大写。

**Comment by liuxybeta** [comment]
您好，您的邮件我已收到，祝您开开心心每一天！

**Comment by atxy-blip** [comment]
参照 `biblatex-gb7714-2015` 手册，给 `biblatex` 包传入 `gbnamefmt=lowercase` 选项就是仅首字母大写。默认设置符合国标规定，可以改，但没必要。

![image](https://user-images.githubusercontent.com/63113546/170213019-23054b4b-93c2-47b5-8339-2c0b088687ca.png)

**Comment by yetiansh** [comment]
可以使用escape，在要保持原样的bib entry出用花括号'{}'括起来

**Comment by LucifierZ** [comment]
> 国标规定如此

非英文字母不会被大写这一般是如何解决的呢？

**Comment by liuxybeta** [comment]
您好，您的邮件我已收到，祝您开开心心每一天！
